.github/workflows/ci.yml

```diff
@@ -507,6 +507,23 @@ jobs:
     - name: run otprotocol test
       run: ./ci/do_ci.ps1 cmake.exporter.otprotocol.test
 
+  windows-build-dll:
+    name: CMake -> exporter proto (Build as DLL)
+    runs-on: windows-2019
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'recursive'
+    - name: setup
+      run: |
+        ./ci/setup_windows_cmake.ps1
+        ./ci/setup_windows_ci_environment.ps1
+        ./ci/install_windows_protobuf.ps1
+    - name: run cmake test (DLL build)
+      run: ./ci/do_ci.ps1 cmake.dll.test
+    - name: run otprotocol test (DLL build)
+      run: ./ci/do_ci.ps1 cmake.exporter.otprotocol.dll.test
```

Comment:
Please define both:

* a DDL build WITH_STL=ON
* a DDL build WITH_STL=OFF

Reply:
> Please define both:
> 
> * a DDL build WITH_STL=ON
> * a DDL build WITH_STL=OFF

The 1st definition will be unnecessary if it is not supported for not (error out).